hooks/usePlayerActions/isAffordable.ts

```diff
@@ -0,0 +1,7 @@
+export default function isAffordable(
+  playerStackSizes: number[],
+  playerIndex: number,
+  betSize: number
+) {
+  return betSize <= playerStackSizes[playerIndex];
```

Comment:
I think `betSize <= playerStackSizes[playerIndex]` is enough readable. Probably you don't need this function 🤔 What do you think?

Reply:
true!